### PR TITLE
fix(operator): cleanup for unused backup jobs and deleted crds in crd-mode

### PR DESCRIPTION
**Pull request by stebenz**
imports https://github.com/caos/orbos/pull/632

### CODE REVIEW

File: operator/database/kinds/databases/managed/adapt.go [hunk 1]
```diff
@@ -264,10 +273,43 @@ func Adapter(
...
+				if desiredKind.Spec.Backups != nil {
+					backupDefs = desiredKind.Spec.Backups
+				}
+				cleanupDestroy, err := cleanup(
+					monitor,
+					backupDefs,
+					k8sClient,
+					namespace,
+					getBackupLabels(),
+				)
```
Comment (eliobischof) [inline operator/database/kinds/databases/managed/adapt.go:286]: into query phase?

**Comment by eliobischof** [comment]
Please import latest ORBOS code

File: operator/database/kinds/databases/managed/adapt.go (hunk 1 above)
Comment (stebenz) [reply operator/database/kinds/databases/managed/adapt.go:286]: yes if you look into the function you would see that the kubeapi has to be queried as to which jobs are currently present

Comment (eliobischof) [reply operator/database/kinds/databases/managed/adapt.go:286]: if i look into the function i can see that it returns destroyers based on a kubeapi query. So it belongs to the query phase, not to the ensure phase?

**Comment by eliobischof** [comment]
@stebenz, can we close this, now that v2 is released?

**Comment by eliobischof** [comment]
not needed anymore, as v2 is released

**Pull request closed without merging**